index.html

```diff
@@ -230,5 +230,6 @@ <h4 class="popup__type">Квартира</h4>
     </div>
   </template>
 
+  <script src="/js/main.js"></script>
```

Comment:
Нужно сделать так, иначе не будет работать в опубликованной версии
```suggestion
  <script src="js/main.js"></script>
```

Reply:
Поправил